### PR TITLE
fix: Send proper configuration

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Send proper configuration

### CODE REVIEW

File: src/RealtimeChannel.ts [hunk 1]
```diff
@@ -212,7 +216,7 @@ export default class RealtimeChannel {
       throw `tried to subscribe multiple times. 'subscribe' can only be called a single time per channel instance`
     } else {
       const {
-        config: { broadcast, presence },
+        config: { broadcast, presence, private: is_private },
```
Comment (filipecabaco) [inline src/RealtimeChannel.ts:219]: private is a reserved word which was causing issues

Comment (soedirgo) [inline src/RealtimeChannel.ts:219]: I think `private`/`is_private` needs to be outside the object, i.e. below `config` and not inside

File: src/RealtimeChannel.ts [hunk 2]
```diff
@@ -197,6 +199,8 @@ export default class RealtimeChannel {
 
     this.broadcastEndpointURL =
       httpEndpointURL(this.socket.endPoint) + '/api/broadcast'
+
+    this.private = this.params.config.private || false
```
Comment (w3b6x9) [inline src/RealtimeChannel.ts:203]: I'm going to remove this. Unused.

File: src/RealtimeChannel.ts (hunk 1 above)
Comment (w3b6x9) [reply src/RealtimeChannel.ts:219]: gonna make into camelCase

**Pull request merged**